### ISSUE TITLE
Drag and drop order in gallery field

### ISSUE THREAD

**Issue by Kraftion** [JoomCoder-com/JoomCCK#208]
Hello
Is it possible in the gallery field to make drag and drop sorting of pictures and also previews in article editing mode?

**Comment by Giorgi625** [comment]
Agree, its 2024 year we should have such features in gallery field.

**Comment by andiamine** [comment]
We will do in future, thanks for suggestions maybe we will use this library
https://github.com/kartik-v/bootstrap-fileinput
